tests/phpunit/integration/Core/Modules/ModulesTest.php

```diff
@@ -472,4 +472,19 @@ public function provider_feature_flag_modules() {
 			$default_modules,
 		);
 	}
+
+	public function test_get_shareable_modules() {
+		$modules = new Modules( new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
+
+		$this->enable_feature( 'dashboardSharing' );
+
+		$shareable_active_modules = array_map( 'get_class', $modules->get_shareable_modules() );
+
+		$this->assertEqualSets(
+			array(
+				'search-console' => 'Google\\Site_Kit\\Modules\\Search_Console',
```

Comment:
@hussain-t looks like we need to update this test to also include the `Google\Site_Kit\Modules\PageSpeed_Insights` module too: https://github.com/google/site-kit-wp/runs/4779620869?check_suite_focus=true